### PR TITLE
Adjust copyright note

### PR DESCRIPTION
**Pull request by ccrisan**
As there's finally a larger group of maintainers that take care of the project, it doesn't make sense anymore to have individual names listed in the copyright note that shows up in UI, especially not mine :)

### CODE REVIEW

**Comment by MichaIng** [comment]
Agreed, makes sense, else the notice grows unlimited 😄. We could make "motionEye" team a link to https://github.com/orgs/motioneye-project/people.

About the AUTHORS file, not sure whether we should maintain this either, since in theory every code contributor should then be added there. Instead, if we want to add credits somewhere, we could link the contributors list on GitHub: https://github.com/motioneye-project/motioneye/graphs/contributors

**Comment by ccrisan** [comment]
> We could also make "motionEye team" a link to https://github.com/orgs/motioneye-project/people.

Great idea, done!

> we could link the contributors list on GitHub: https://github.com/motioneye-project/motioneye/graphs/contributors

Also a good idea, now `AUTHORS` points the reader to GitHub's contributos.

File: motioneye/templates/main.html [hunk 1]
```diff
@@ -1257,7 +1257,7 @@
         <img class="background-logo" src="{{static_path}}img/motioneye-logo.svg" onmousedown="return false;">
         <div class="page-container"></div>
         <div class="footer">
-            <div class="copyright-note">copyright &copy; The motionEye Team</div>
+            <div class="copyright-note">copyright &copy; <a href="https://github.com/orgs/motioneye-project/people">The motionEye Team</a></div>
```
Comment (MichaIng) [inline motioneye/templates/main.html:1260]:
```suggestion
            <div class="copyright-note">copyright &copy; <a href="https://github.com/orgs/motioneye-project/people" target="_blank" rel="noopener">The motionEye Team</a></div>
```
Making this a link in a new browser tab 🙂.

**Review by MichaIng — APPROVED** [review]

**Comment by Fabeson** [comment]
Hello all,

not sure where else to contact the maintainers, so sorry in advance if this is not the right place for this (kinda new to github and not a dev myself ;))

Does this recent activity mean the hand-over is taking place and you guys are gonna revive MotionEyeOS? That would be amazing!

**Comment by MichaIng** [comment]
Yes it is. But there is no ETA for when motionEyeOS will be updated, first of all we need to have a stable motionEye release.

**Comment by Fabeson** [comment]
That's great news, thanks for the quick reply @MichaIng!

I will have an eye on this repository, looking forward to the updates and hope I can contribute later down the line with testing and bug reports :)

Cheers and have a nice weekend!

**Comment by MichaIng** [comment]
> hope I can contribute later down the line with testing and bug reports :)

That would be great, every help is highly welcome. Have a nice weekend, too!

**Pull request merged**